deploy/operator.antrea.vmware.com_v1_antreainstall_cr.yaml

```diff
@@ -84,6 +84,9 @@ spec:
     # the flow collector.
     # Flow export frequency should be greater than or equal to 1.
     #flowExportFrequency: 12
+
+    # Provide the service ip range(--service-cluster-ip-range).
+    serviceCIDR: "10.96.0.0/12"
```

Comment:
add a `#` at the beginning of the line?

Reply:
> add a `#` at the beginning of the line?

Thanks, and for these manifest files under directory `deploy`, I'll push a new commit to divide them into two parts(`deploy/kubernetes` and `deploy/openshift`) in this patch.